docs/fundamentals/write-operations.txt

```diff
@@ -480,3 +479,134 @@ with ``"contemporary"`` in the ``genres`` array field. Click the
+
+To learn about the Laravel features available in {+odm-short+} that modify
+delete behavior, see the following sections:
+
+- :ref:`Soft delete <laravel-model-soft-delete>`, which lets you mark
+  documents as deleted instead of removing them from the database
+- :ref:`Pruning <laravel-model-pruning>`, which lets you define conditions
+  that qualify a document for automatic deletion
+
+.. _laravel-fundamentals-delete-one:
+
+Delete a Document Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can delete one document in the following ways:
+
+- Call the ``destroy()`` method on the model, passing it the primary key of
+  the document to be deleted.
+- Call the ``delete()`` method on an instance of the model.
+- Chain methods to retrieve and delete an instance of a model by calling the
+  ``delete()`` method.
+
+The following example shows how to delete a document by passing the value of
+its primary key to the ``destroy()`` method:
+
+.. literalinclude:: /includes/fundamentals/write-operations/WriteOperationsTest.php
+   :language: php
+   :dedent:
+   :start-after: begin model delete by id
+   :end-before: end model delete by id
+   :caption: Delete the document by its primary key.
+
+When the ``destroy()`` method succeeds, it returns the number of documents
+deleted.
+
+If the primary key value does not match any documents, the ``destroy()`` method
+returns returns ``0``.
+
+The following example shows how to delete a document by calling ``delete()``
+on an instance of the model:
+
+.. literalinclude:: /includes/fundamentals/write-operations/WriteOperationsTest.php
+   :language: php
+   :dedent:
+   :start-after: begin delete one model
+   :end-before: end delete one model
+   :caption: Delete the document by calling the delete() method on an instance.
+
+When the ``delete()`` method succeeds, the operation returns the number of
+documents deleted.
+
+If the retrieve part of the call does not match any documents in the collection,
+the operation returns ``0``.
+
+The following example shows how to chain calls to retrieve the first
+matching document and delete it:
+
+.. literalinclude:: /includes/fundamentals/write-operations/WriteOperationsTest.php
+   :language: php
+   :dedent:
+   :start-after: begin model delete one fluent
+   :end-before: end model delete one fluent
+   :caption: Delete the matching document by chaining the delete() method.
+
+.. include:: /includes/fact-orderby-id.rst
+
+When the ``delete()`` method succeeds, it returns the number of documents
+deleted.
+
+If the ``where()`` method does not match any documents, the ``delete()`` method
+returns returns ``0``.
+
+.. _laravel-fundamentals-delete-many:
+
+Delete Multiple Documents Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can delete multiple documents in the following ways:
+
+- Call the ``destroy()`` method on the model, passing the primary keys of
+  the documents to be deleted.
+  keys to delete the matching documents.
```

Comment:
This sentence seems incomplete. `Model::destroy($ids)` accepts a list of ids, or a list of model instances.

Note that using this method with a large number of ids is unefficient as it sends a distinct command for each id. For performance it's better using `Model::whereIn('id', $ids)->delete()`

Reply:
Good catch. I'll add an admonition where the example is currently placed to mention the performance issue.